![DNA](res_brcf_bioinformatics_dna_stock_blue.jpeg)
# University of Michigan Bioinformatics Core

We help researchers identify and interpret patterns in RNA, DNA, and proteins by placing high-throuhgput data into a biologically meaningful context. This includes:

- **Elaborating experimental design**
- **Developing reproducible workflows**
- **Analyzing next-generation sequencing data**
- **Assisting in visualizing and interpreting results**
- **Letters of support, grant proposal development, or manuscript revision**
- **Hosting virtual, hands-on bioinformatics workshops** (see [current workshop schedule and descriptions](https://michmed.org/XYQwq))

## We would love to talk with you

See [our website](https://michmed.org/GqGzZ) for more information about the core staff or reach out directly at bioinformatics@umich.edu to [set up a free consulation](https://docs.google.com/forms/d/e/1FAIpQLSepk7VqOl3xmBgkZybrl71VuQmKk3YmkgmpaBO4dD2hOtIh4w/viewform).

## Bioinformatics analyses we support

We have expertise in a range of bioinformatics analyses, including:

- **Bulk RNA-Seq gene expression**

    - Poly(A) selection, total RNA, miRNA, Ribo-Seq, long-read gene expression
    - Differential gene expression
    - Differential isoform / isoform switching 
    - Functional enrichment analysis (GO terms, KEGG pathways)
    - [Workshop: RNA-Seq Demystified](https://medresearch.umich.edu/office-research/about-office-research/biomedical-research-core-facilities/bioinformatics-core/bioinformatics-workshops-training#rna-seq-demystified)
    - [Sample RNA-Seq analysis report](https://umich-brcf-bioinf.github.io/Watermelon/doc/SampleReport.html)
    - Tools: [nf-core/rnaseq analysis pipeline](https://nf-co.re/rnaseq) | [DESeq2](https://bioconductor.org/packages/devel/bioc/vignettes/DESeq2/inst/doc/DESeq2.html) | [iPathwayGuide](https://advaitabio.com/bioinformatics/ipathwayguide/) | [WebGestalt](https://www.webgestalt.org/) | [GSEA](https://www.gsea-msigdb.org/gsea/index.jsp)

- **Single-cell/single-nuclei analysis**
 
    - 3' or 5' scRNA-Seq gene expression
    - V(D)J immune profiling
    - scATAC-Seq
    - Trajectory analysis
    - Single-cell analysis of long-reads
    - [Workshop: Intro to Single-Cell Analysis](https://medresearch.umich.edu/office-research/about-office-research/biomedical-research-core-facilities/bioinformatics-core/bioinformatics-workshops-training#intro-to-single-cell-analysis)
    - Tools: [Cell Ranger](https://www.10xgenomics.com/support/software/cell-ranger/latest)] | [Seurat](https://satijalab.org/seurat/) | [scCatch](https://github.com/ZJUFanLab/scCATCH)

- **Spatial sequencing analysis**

    - Visium, Xenium, GeoMX DSP
    - Tools: [Space Ranger](https://www.10xgenomics.com/support/software/space-ranger/latest) | [Seurat](https://satijalab.org/seurat/) | [Xenium Explorer](https://www.10xgenomics.com/support/software/xenium-explorer/latest) | [GeoMX tools](https://www.bioconductor.org/packages/release/workflows/vignettes/GeoMxWorkflows/inst/doc/GeomxTools_RNA-NGS_Analysis.html)

- **Variant identification / assembly**

    - Variant identification / structural variation from WGS, exome, panel, long-reads
    - Copy Number Analysis from WGS
    - Genome assembly from short reads, long-reads, hybrid
    - Tools: [nf-core/sarek](https://nf-co.re/sarek) | [VEP](https://www.ensembl.org/info/docs/tools/vep/index.html) | [SnpEff](https://pcingola.github.io/SnpEff) | [SPAdes](https://github.com/ablab/spades) | [Velvet](https://github.com/dzerbino/velvet) | [Flye](https://github.com/mikolmogorov/Flye) 

- **Epigenomic analysis**

    - DNA Methylation from WGBS/oxBS/EM-Seq, ERRBS/oxERRBS, long-reads
    - Chromatin accessibility from bulk ATAC-Seq
    - Histone profiling from ChIP-Seq / Cut & Run / Cut & Tag
    - Transcription factor binding from ChIP-Seq
    - EPIC-Array
    - Tools: [nf-core/methylseq](https://nf-co.re/methylseq) | [nf-core/atacseq](https://nf-co.re/atacseq) | [nf-core/chipseq](https://nf-co.re/chipseq) | [nf-core/cutandrun](https://nf-co.re/cutandrun)

- **Microbiome analysis**

    - 16S amplicon
    - metagenomics / metatranscriptomics
    - Tools: [mothur](https://mothur.org/) | [SqueezeMeta](https://github.com/jtamames/SqueezeMeta)

- **Proteome expression**

    - Liquid Chromatography-Mass Spectrometry
    - Antibody/Aptamer (ELISA, OLink, Somalogic)

